ansible/roles/es-mapping/files/mappings/userv3-mapping.json

```diff
@@ -889,6 +889,937 @@
           }
         }
       }
+    },
+    "profileDetails": {
+      "properties": {
+        "photo": {
+          "type": "text",
+          "fields": {
+            "raw": {
+              "type": "text",
+              "analyzer": "keylower",
+              "fielddata": true
+            }
+          },
+          "copy_to": [
+            "all_fields"
+          ],
+          "analyzer": "cs_index_analyzer",
+          "search_analyzer": "cs_search_analyzer",
+          "fielddata": true
+        },
+        "personalDetails": {
+          "properties": {
+            "primaryEmail": {
+              "type": "text",
```

Comment:
Do we need this email field as text , Do we really need this field to be analyzed or apply fuzzy or auto search capability